nix/zig-libxml2-src.nix

```diff
@@ -0,0 +1,13 @@
+{ lib, fetchFromGitHub }:
```

Comment:
This whole thing could just be put in a

`let ... in` at the top of `package.nix` instead if you prefer